test/Quake-QIR/return_values.qke

```diff
@@ -6,7 +6,7 @@
 // the terms of the Apache License 2.0 which accompanies this distribution.   //
 // ========================================================================== //
 
-// RUN: cudaq-opt --add-dealloc --kernel-execution --canonicalize %s | \
+// RUN: cudaq-opt --add-dealloc --kernel-execution=codegen=1 --canonicalize %s | \
```

Comment:
We should add a work item to extend tests like this to check both the `codegen=1` and `codegen=0` outputs. The longer range plan to deprecate the `codegen=1` option.